keyspace.go

```diff
@@ -124,10 +124,14 @@ func (k *k) MultiTimeSeriesTable(name, indexField, timeField, idField string, bu
 		panic("Unrecognized row type")
 	}
 	m[bucketFieldName] = time.Now()
+	clusteringColumns := []string{timeField}
+	if idField != timeField {
+		clusteringColumns = append(clusteringColumns, idField)
+	}
 	return &multiTimeSeriesT{
 		t: k.NewTable(fmt.Sprintf("%s_multiTimeSeries_%s_%s_%s_%s", name, indexField, timeField, idField, bucketSize.String()), row, m, Keys{
 			PartitionKeys:     []string{indexField, bucketFieldName},
-			ClusteringColumns: []string{timeField, idField},
+			ClusteringColumns: clusteringColumns,
```

Comment:
This bug doesn't happen with Gocassa's in memory mocks, this is why I can't replicate it in a unit test.

I'm not sure what our goal is for these mocks: Do we want it to be the most accurate reflection of Cassandra possible? If so, I could bake in this error case, so if we try to issue an `UPDATE` with duplicated relations, we reject that update?